2022/34xxx/CVE-2022-34872.json

```diff
@@ -1,18 +1,70 @@
 {
```

Comment:
'https://docs.centreon.com/docs/21.10/releases/centreon-core/\r\n\r\nhttps://github.com/centreon/centreon/blob/master/SECURITY_ACK.md#-2022-' does not match '^(ftp|http)s?://\\S+$'